routers/api/v1/api.go

```diff
@@ -433,17 +433,16 @@ func RegisterRoutes(m *macaron.Macaron) {
 		// Repositories
 		m.Post("/org/:org/repos", reqToken(), bind(api.CreateRepoOption{}), repo.CreateOrgRepo)
 
-		m.Group("/repos", func() {
-			m.Get("/search", repo.Search)
-		})
-
 		m.Combo("/repositories/:id", reqToken()).Get(repo.GetByID)
 
 		m.Group("/repos", func() {
+			m.Get("/search", repo.Search)
```

Comment:
This change should not be in this PR. Maybe put it a separate PR?

Reply:
I removed that change from this commit. Will submit a new PR for it.